### PR TITLE
Theming#399

### PR DESCRIPTION
**Pull request by Ardeshir81**
HI! This fixes issue #399 + minor theming tweaks

### CODE REVIEW

File: client/app/src/accounts/account.controller.js [hunk 1]
```diff
@@ -219,7 +219,7 @@
     if (!self.network.themeDark) self.network.themeDark = false
 
     // will be used in view
-    self.currentTheme = self.network.theme
+    self.currentTheme = 'default';//self.network.theme
```
Comment (spresnal) [inline client/app/src/accounts/account.controller.js]: Should probably remove the commented out code here

Comment (Ardeshir81) [reply client/app/src/accounts/account.controller.js]: Thanks @spresnal . But setting theme's name before registering it caused a lot of extra watches. watching themes and colors in angularJS Material is so expensive and inefficient 
screenshot from 2017-11-16 23-08-42
BTW, Thanks for your hint, I made some other changes, if they didn't fix it, I'll revert them. (just preserving the main fix commit)

THanks :)

**Comment by Ardeshir81** [comment]
@j-a-m-l Thanks for your review. I actually couldn't find out what the error you told me was about : `ReferenceError: vShare is not defined`
that `vShare` I think is an iOS thing. I'm not sure what that was about and I couldn't reproduce it, but yeah, my commits caused more issues. I added two more commits. Please check if what you did happens with these new commits.
Or if possible please tell me how to reproduce the issue . Thanks : )

BTW,that `vShare` .. were you thinking `bout something else while writing that review ? : )

**Comment by fix** [comment]
i have the vshare issue here with latest from master, on macOs

**Comment by fix** [comment]
@j-a-m-l did you fix the vshare issue?

**Comment by Ardeshir81** [comment]
@fix I couldn't reproduce issue. So I, no! Didn't

**Comment by j-a-m-l** [comment]
I haven't had that `vShare` problem again.

**Comment by boldninja** [comment]
@Ardeshir81 can you provide me with your ARK address?

**Comment by Ardeshir81** [comment]
@boldninja yeah . can I ask why ? do you want it here ? or somewhere else ?

**Comment by boldninja** [comment]
@Ardeshir81 you can contact me on Slack if you don't want it public. Its to receive github bounty.

**Comment by Ardeshir81** [comment]
Thank you @boldninja . I'll contact you in slack

**Pull request merged**